tests/test_core.py

```diff
@@ -125,12 +125,12 @@ def test_create_fake_users(db):
 
 
 def test_create_address(db):
-    assert Address.objects.all().count() == 0
+    assert not Address.objects.exists()
     random_data.create_address()
     assert Address.objects.all().count() == 1
```

Comment:
```suggestion
    assert Address.objects.all().count() == counter + 1
```

This way test is not depending on empty database.

Reply:
I don't like this. That way we are missing information whether other conftest are adding additional data. Also checking if an object is saved with proper data will require additional check - if there is already something in the database or not.

Reply:
When maintaining forks we are sometimes creating preexisting objects in database (for example main company address). If we add such fixture, then all tests with 'frozen' counters will fail.

Reply:
I'm not sure I follow - you are adding objects in migrations, right? Are they required in migrations or could be replaced with fixture used with `Django admin`? Is there any discussion about that so I could read more about this issue?
I understand your point, but I also see other issues. For eg 
- what should happen when we want to check if proper data are saved to DB? Couse `Model.objects.first()` might fail. 
- what if we have a field with `unique=True` and we have a fixture with `autouse=True` that sets this field to the same value as we set it in our test? We've got an error and it will take additional time to find why this is happening.